backend/de.metas.marketing/base/src/main/java/de/metas/marketing/base/spi/PlatformClient.java

```diff
@@ -40,7 +40,8 @@
 
 	List<RemoteToLocalSyncResult> syncCampaignsRemoteToLocal(List<Campaign> campaigns);
```

Comment:
it seems that when exchanging data with the CMS-platform, we pretty much rely on having enough memory.
- do the platforms have pagination mechanisms that we use?
- do we split the data that we want to send in chunks?